vault/activity_log.go

```diff
@@ -1737,13 +1737,11 @@ func modifyResponseMonths(months []*ResponseMonth, start time.Time, end time.Tim
 	}
 	start = timeutil.StartOfMonth(start)
 	if timeutil.IsCurrentMonth(end, time.Now().UTC()) {
-		end = timeutil.StartOfMonth(end).AddDate(0, -1, 0)
+		end = timeutil.EndOfMonth(timeutil.StartOfMonth(end).AddDate(0, -1, 0))
 	}
-	end = timeutil.EndOfMonth(end)
```

Comment:
do we still need to pin `end` to be the end of the provided `end` month in the case where `end` is not the current month?

e.g. what if `end` is in the middle of last month?

Reply:
is "provided" end month here referencing the `end_date` query param? If so, in the UI we send the 20th of the month, as we expect the backend to re-adjust it to the last day of that month.

Reply:
following up, appears to be something different - as the response looks correct on my end. disregard!

Reply:
Fixed!